compiler/rustc_passes/src/weak_lang_items.rs

```diff
@@ -59,7 +59,7 @@ fn verify<'tcx>(tcx: TyCtxt<'tcx>, items: &lang_items::LanguageItems) {
         }
     }
 
-    for (name, &item) in WEAK_ITEMS_REFS.iter() {
```

Comment:
`WEAK_ITEMS_REFS` is already a `StableMap` now, do you need to sort it? The only reason I am commenting on it is that I am slightly saddened by the new `.clone()` (but I'll get over it).

Reply:
This PR is the one turning it into a `StableMap`, and the sort is necessary to normalize the order, right?

(That is: If we don't normalize the order, then we don't get the benefit of this change, right?)

To be clear: I too am saddened by the `.clone()`, and you might talk me into just using a different structure entirely. This seemed like the simplest fix, and if we're going to balk at doing it in this scenario, then maybe we should remove `StableMap` entirely and figure out a different way to discourage people from using `FxHashMap`?